### PR TITLE
V2 to V3 librange support

### PR DESCRIPTION
**Pull request by gskchaitanya**

### CODE REVIEW

File: core/src/main/java/net/opentsdb/query/pojo/TagVFilter.java [hunk 1]
```diff
@@ -660,6 +660,14 @@ public TagVFilter build() {
             "The tagk cannot be null or empty");
       }
       
+      // V2 librange conversion
+      if (type.equals(TagVLiteralOrFilter.FILTER_NAME)) {
```
Comment (manolama) [inline core/src/main/java/net/opentsdb/query/pojo/TagVFilter.java:664]: Doesn't Olympus actually set the `type` to `librange`? I thought it converted properly.

Comment (gskchaitanya) [reply core/src/main/java/net/opentsdb/query/pojo/TagVFilter.java:664]: Olympus does set it to librange, but the API that's being used by the touchstone/deploy builds doesn't convert those.

File: core/src/main/java/net/opentsdb/query/pojo/TagVFilter.java [hunk 2]
```diff
@@ -660,6 +660,14 @@ public TagVFilter build() {
...
+        if (filter.startsWith("librange://@")) {
```
Comment (anonthing) [inline core/src/main/java/net/opentsdb/query/pojo/TagVFilter.java:665]: can this be handled in our plugin?

Comment (gskchaitanya) [reply core/src/main/java/net/opentsdb/query/pojo/TagVFilter.java:665]: Yes, but it won't be clean. Several classes need to be extended and will introduce a bit more plugins as we discussed.

**Review by anonthing — APPROVED** [review]

**Pull request merged**